README.md

```diff
@@ -21,4 +27,146 @@ dotnet test
 ```
 dotnet test --collect:"XPlat Code Coverage"
 dotnet reportgenerator -reports:"**/*.cobertura.xml" -targetdir:".coveragereport"
+```
+
+## Conventions
```

Comment:
Can we move this to a separate README-CODE-CONVERTIONS.md file? the content of this section is potentially big and could continue growing, it might deserve its own file.

This is good, I would like to have comments from the team on these conventions, but as far as I see, those make sense with default conventions documented in C# reference: https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions

@adrielgamorim , @svaleria , @Schlvf , @Luisjor , please take a look and make sure to add any comment you have about these conventions